### DISCUSSION TITLE
Animax Japan (English Subs)

### DISCUSSION THREAD

**Discussion by Frizzy1312** [iptv-org/community#740] [Q&A]
the Link for animax Japan English subs from the Japanese country pack and animation category pack is broken and is not opening 

 

**_Channel Name:_** xxx Animax Japan (English sub) (1080p)
**_Channel ID:_** Animax.jp
**_Country:_** xxx Japan pack (watching from india) 
**_Broken Link (from playlist):_** xxx #EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-country="JP" tvg-language="Japanese;English" tvg-logo="https://i.imgur.com/zpi6mQ3.png" group-title="Animation",Animax Japan (English Subs) (1080p)
https://feed.play.mv/live/10005200/isU48qPZqV/master.m3u8
**_Possible Replacement (optional):_** xxx
**_Notes (optional):_** xxx

**Comment by Dum4G** [comment]
![image](https://user-images.githubusercontent.com/5430116/143958462-65fdde2b-2af0-4622-a052-ff4cf10e968e.png)
Link may come back in forseeable future

**Comment by Klpra** [comment]
Brother any idea for watching animax English subtitles channel online it's really helpful for me and animax will be add future?

**Comment by psycoball** [comment]
i want link too

**Comment by xinomp** [comment]
Any update on an English stream link?

**Comment by Klpra** [comment]
Not yet; but you can watch animax Eng sub (sd) quality on a IPTV mac portel

**Comment by PopEDMRap** [comment]
Where can I watch Animax or Aniplus in English please?

**Comment by xinomp** [comment]
> Not yet; but you can watch animax Eng sub (sd) quality on a IPTV mac portel

How can I do that?

**Comment by OTKxRAGE** [comment]
Any working link for Animax (English Sub) or AniPlus (English Sub) ?

**Comment by drmrman35** [comment]
Is it possible to extract it from mewatch.sg? Other channels from the website have been extracted before
https://github.com/iptv-org/iptv/issues/1983

**Comment by Waterfall14** [comment]
Can you guys please add Animax (English Sub) and Aniplus (English Sub) to your playlist?

you can extract these channels from one of these tv providers: https://www.animax-asia.com/get-animax

https://www.singtel.com

**Comment by Shadowface96** [comment]
Anyone can add Animax (English Sub) to the playlist? I cannot understand vietnamese

**Comment by RiZe370z** [comment]
So its not possible to watch animax?

**Comment by MeanXBeam** [comment]
i saw that mewatch singapore has animax and gem if you guys could get it probably would have english subtitles

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19734
